test/Raven/Tests/ClientTest.php

```diff
@@ -2072,7 +2072,7 @@ public function testEncode()
         $this->assertRegExp('_^[a-zA-Z0-9/=]+$_', $value, 'Raven_Client::encode returned malformed data');
         $decoded = base64_decode($value);
         $this->assertInternalType('string', $decoded, 'Can not use base64 decode on the encoded blob');
-        if (function_exists("gzcompress")) {
+        if (function_exists('gzcompress') && extension_loaded('zip')) {
```

Comment:
Another case where I think zip extension doesn't matter?

Reply:
It's the same stuff, but on the other side, the test... I'll try to change this and see if it still works